pytorch/training/docker/2.0/py3/Dockerfile.cpu

```diff
@@ -107,7 +108,9 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
  && rm ~/mambaforge.sh
 
 # install core conda dependency packages (and pin essential packages)
-RUN /opt/conda/bin/mamba install -y -c conda-forge \
+RUN /opt/conda/bin/mamba install -y --override-channels \
+    -c https://aws-ml-conda-ec2.s3.us-west-2.amazonaws.com \
```

Comment:
what packages are we pulling from the conda channel?

Reply:
For this install, no. But if anything can be pulled from that channel it uses that channel which has more priority. If we add it, then it avoid further conflicts on installation commands

Reply:
if there are no packages, then adding this seems to be redundant, we are anyways using this channel explicitly in the below RUN commands, we can use it for new packages if so